SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -0,0 +1,2895 @@
+            solver->SetCPressureTarget(iMarker, iVertex, 0.0);
+          }
+          
+        }
+      }
+    }
+    
+    getline(Surface_file, text_line);
+    
+    while (getline(Surface_file, text_line)) {
+      for (icommas = 0; icommas < 50; icommas++) {
+        position = text_line.find( ",", 0 );
+        if (position!=string::npos) text_line.erase (position,1);
+      }
+      stringstream  point_line(text_line);
+      
+      if (geometry->GetnDim() == 2) point_line >> iPoint >> XCoord >> YCoord >> Pressure >> PressureCoeff;
+      if (geometry->GetnDim() == 3) point_line >> iPoint >> XCoord >> YCoord >> ZCoord >> Pressure >> PressureCoeff;
+      
+      if (PointInDomain[iPoint]) {
+        
+        /*--- Find the vertex for the Point and Marker ---*/
+        
+        iMarker = Point2Vertex[iPoint][0];
+        iVertex = Point2Vertex[iPoint][1];
+        
+        solver->SetCPressureTarget(iMarker, iVertex, PressureCoeff);
+        
+      }
+      
+    }
+    
+    Surface_file.close();
+    
+    delete [] Point2Vertex;
+    delete [] PointInDomain;
+    
+    /*--- Compute the pressure difference ---*/
+    
+    PressDiff = 0.0;
+    for (iMarker = 0; iMarker < config->GetnMarker_All(); iMarker++) {
+      Boundary   = config->GetMarker_All_KindBC(iMarker);
+      
+      if ((Boundary == EULER_WALL             ) ||
+          (Boundary == HEAT_FLUX              ) ||
+          (Boundary == ISOTHERMAL             ) ||
+          (Boundary == NEARFIELD_BOUNDARY)) {
+        for (iVertex = 0; iVertex < geometry->GetnVertex(iMarker); iVertex++) {
+          
+          Normal = geometry->vertex[iMarker][iVertex]->GetNormal();
+          
+          Cp = solver->GetCPressure(iMarker, iVertex);
+          CpTarget = solver->GetCPressureTarget(iMarker, iVertex);
+          
+          Area = 0.0;
+          for (iDim = 0; iDim < geometry->GetnDim(); iDim++)
+            Area += Normal[iDim]*Normal[iDim];
+          Area = sqrt(Area);
+          
+          PressDiff += Area * (CpTarget - Cp) * (CpTarget - Cp);
+        }
+        
+      }
+    }
+    
+#ifdef HAVE_MPI
+    su2double MyPressDiff = PressDiff;    
+    SU2_MPI::Allreduce(&MyPressDiff, &PressDiff, 1, MPI_DOUBLE, MPI_SUM, MPI_COMM_WORLD);
+#endif
+    
+  }
+  
+  /*--- Update the total Cp difference coeffient ---*/
+  
+  solver->SetTotal_CpDiff(PressDiff);
+  
+  SetHistoryOutputValue("CP_DIFF", PressDiff);
+
+}
+
+void CFlowOutput::WriteAdditionalFiles(CConfig *config, CGeometry *geometry, CSolver **solver_container){
+  
+  if (config->GetFixed_CL_Mode() || config->GetFixed_CM_Mode()){
+    WriteMetaData(config, geometry);
+  }
+  
+  if (config->GetWrt_ForcesBreakdown()){
+    WriteForcesBreakdown(config, geometry, solver_container);
+  }
+  
+}
+
+void CFlowOutput::WriteMetaData(CConfig *config, CGeometry *geometry){
+    
+  ofstream meta_file;
+  
+  string filename = "flow";
+  
+  filename = config->GetFilename(filename, "meta", curTimeIter);
```

Comment:
Here the meta data is written to `flowmeta`, but when the solver tries to read the meta data, the file it tries to read is `flow.meta`. I like the name `flow.meta` better, but it doesn't matter much as long as it is consistent

Reply:
Oops thats a typo. Thought I'd solved that already. Thanks!